### DISCUSSION TITLE
implementation to acoustics.

### DISCUSSION THREAD

**Discussion by makeabhishek** [CERN/TIGRE#447] [General]
Is it possible to implement TIGRE for acoustic/ultrasound tomography? 
Any suggestions

**Comment by AnderBiguri** [comment]
Hi @makeabhishek . As it comes, TIGRE only has CT projectors. If you have your own forward and backprojector for acustic/ultrasound, then you can replace TIGRE's to make it happen. However, if you don't you won't be able to do it no, there is no inmediate plan to add it in the future either.
